widget/button.go

```diff
@@ -309,25 +334,15 @@ func (r *buttonRenderer) buttonColor() color.Color {
 	switch {
 	case r.button.Disabled():
 		return theme.DisabledButtonColor()
+	case r.button.focused:
+		return blendColor(theme.ButtonColor(), theme.FocusColor())
```

Comment:
This should be the corresponding bg color depending on the importance, however I agree that the color resulted after blending with the primary color does not have any appreciable difference, maybe this could be improved if we decide in the future to change the Focus color (to something similar to MaterialDesign), this should be needed if we want different colored buttons (primary, secondary, success, danger, etc). However, for now I think it is ok :)

Reply:
Yes I wondered that as well, but it just didn't seem to look good. Maybe we can revisit it somehow, like if the focus was not a directly less-vibrant version of primary.